doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1234,10 +1234,30 @@ Controlling Typing Flags
    :cmd:`Print Assumptions`. It has the same effect as `-type-in-type` command line
    argument (see :ref:`command-line-options`).
 
+.. flag:: Conversion Checking
+
+   This flag can be used to enable/disable checking of conversion
+   problems. Conversion tests which infer universe constraints (e.g.
+   as used by elaboration) is not affected. This breaks the
+   consistency of the system and risks unsafe memory accesses (through
+   VM and native compute). Note that `coqchk` ignores this flag and
+   always checks conversions.
+
+   For instance :g:`0 : bool` is accepted. When a specific shape is
+   expected (match discriminee needs to have an inductive type, binder
+   annotation must be have a sort type, application head must have a
+   function type) an explicit cast may be needed. For instance :g:`0
+   0` is not accepted, but `(0 : bool -> bool) 0` is accepted.
+
+   Since the upper layer checks are not affected, you may need to work
+   around them (for instance using :tacn:`exact_no_check`) to make use
+   of this flag.
+
 .. cmd:: Print Typing Flags
 
-   Print the status of the three typing flags: guard checking, positivity checking
-   and universe checking.
+   Print the status of the typing flags: kernel checks, guard
+   checking, positivity checking, universe checking, template
+   polymorphism checking and `indices-matter`.
 
```

Comment:
The flags should use the proper names and be hyperlinked to "Guard Checking", "Positivity Checking", "Universe Checking", etc.

Perhaps you would also make the printed output use the official flag names.

![image](https://user-images.githubusercontent.com/1253341/73985396-e4216980-48ef-11ea-85f1-75dec0b7e1df.png)

I would also wonder whether "Print Typing Flags" is really needed.  A user can just do "Test Positivity Checking", etc. for each flag.